examples/Attributes/publications.md

```diff
@@ -92,22 +124,24 @@ as a list in the `value` field, as below:
 }
 ```
 
-5. If a source provides a CURIE identifier for a supporting publication, Knowledge Providers MUST ensure the 
+3. If a source provides a CURIE identifier for a supporting publication, it MUST follow the
 prefix spelling and casing match that in the Biolink Model [prefix map](https://github.com/biolink/biolink-model/blob/master/prefix-map/biolink-model-prefix-map.json). (e.g "PMID:1593752", "doi:10.1177/00928615010300134").
 
-6. If a source provides a URL for a supporting publication, Knowledge Providers MUST report this full URL directly, 
-EXCEPT where a Pubmed, Pubmed Central, or DOI identifier is provided as part of a full URL. In such cases, KPs 
-MUST report the CURIE form of this identifier, e.g.:  
-    a. http://www.ncbi.nlm.nih.gov/pubmed/29076384 → PMID: 29076384  
-    b. http://europepmc.org/articles/PMC6246007 → PMC:6246007  
-    c. https://doi.org/10.1080/17512433.2018.1398644 → DOI:0.1080/17512433.2018.1398644  
 
-7. Where multiple types of publication identifiers or URLs exist for a single Publication (e.g. a PMID, PMCID, 
+4. A source MAY provide a URL as one of the values of the `biolink:publications` attribute, EXCEPT in cases where  
+a Pubmed, Pubmed Central, or DOI identifier is part of a full URL. In such cases, the identifier MUST be reported 
+in its CURIE form:  
+    a. http://www.ncbi.nlm.nih.gov/pubmed/29076384   →  **PMID: 29076384**  
+    b. http://europepmc.org/articles/PMC6246007      →  **PMC:6246007**  
+    c. https://doi.org/10.1080/17512433.2018.1398644 →  **DOI:0.1080/17512433.2018.1398644**  
+
+
+5. Where multiple types of publication CURIEs exist for a single Publication (e.g. a PMID, PMCID, 
```

Comment:
We should clarify the intent of this instruction. I had a couple things in mind:
 
1. If a source provides multiple identifiers for the same pub, which I think some will do - KPs should report only one of them - in order of precedence: PMID, PMCID, DOI, bespoke identifier.

2. If a source provides only a PMCID, or DOI - do we instruct the KP to try to find a PMID for the publication and report this where it exists?

Reply:
@cbizon @gaurav  - does NodeNorm return a normalized identifier for Publication?

Reply:
No, it doesn't, though I suppose it could if that were useful....